explainer.md

```diff
@@ -102,6 +120,8 @@ if ('launchQueue' in window) {
 ```
 An application could then choose to handle these files however it chose. For example, it could save the file handle to disk and create a URL to address the launched file, allowing users to navigate back to a file.
 
+The `launchParams` property is discussed in more detail in the [Launch Events](https://github.com/WICG/sw-launch/issues/20) explainer.
```

Comment:
This text implies the link points to the launch events explainer, but the link points to an issue in the launch events explainer repo, instead.  Also, the text itself 'launchParams' is only mentioned briefly so I imagine it will be hard to pick up the meaning quickly by someone just wanting some more info on that topic.

What should I be picking up from that issue that you think one wouldn't get from the launch events explainer itself?  I'm wondering if we should get the SWLE explainer updated at some point with that if so.

Reply:
Ah yeah, this text was already in the explainer, so I just moved it around. I agree that the link is a bit confusing but just didn't want to remove the potentially useful info. Not sure there's too much that's useful for an explainer in that link, as it's mostly more detailed API design discussion, but figured we could leave it there in case folks are curious. WDYT?

Reply:
I tried to clarify the text a bit. Please lmk if you'd like to remove the link (and this sentence) altogether